crates/runners/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "gadget-runners"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dependencies]
+gadget-runner-core = { workspace = true }
+gadget-runner-eigenlayer = { workspace = true, optional = true }
+gadget-runner-symbiotic = { workspace = true, optional = true }
+gadget-runner-tangle = { workspace = true, optional = true }
+
+
+[features]
+default = ["std"]
+
+std = [
+    "gadget-runner-core/std",
+#    "gadget-runner-eigenlayer?/std",
+#    "gadget-runner-symbiotic?/std",
+#    "gadget-runner-tangle?/std"
+]
+no-std = [
+#    "gadget-runner-core/no-std",
+#    "gadget-runner-eigenlayer?/no-std",
```

Comment:
I don't imagine you'll need these. The default features are off. The only reason for specifying no_std feature is when a package like `subxt` has two modes `native` and `web`. I think most deps from `alloy` are default no_std compatible.